v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py

```diff
@@ -130,6 +138,8 @@ def test_update_vat_with_updated_rdc_snv_indel_38(
                 updates=hl.empty_set(hl.tstruct(callset=hl.tstr, project_guid=hl.tstr)),
             ),
         )
+        mock_get_datasets_to_update.return_value = []
```

Comment:
I'm a little confused by this mock too.  We set `datasets_to_update` to be the empty list, but the rest of the test includes updating [all the datasets](https://github.com/broadinstitute/seqr-loading-pipelines/pull/718/files#diff-d7e252ad0022625cd78f557ba25857241d79b35a8a6d9a99b32daa1672b7385cR165-R182)

Reply:
These tests begin with a nonexistent annotations table, so  on the first `complete()` call, `initialize_table()` is called then `update_table()` adds all the datasets to the vat.  

During the next `complete()` call, if this were real life, `get_datasets_to_update()` should return an empty list

Reply:
maybe I need a test case which also updates an existing vat